electron/app/chatBro/db/index.ts

```diff
@@ -1,12 +0,0 @@
-import * as _ from 'lodash';
```

Comment:
I thought it made more sense if this was in the init functions folder because these are shared and not specific to chat.db

Reply:
JK in my second PR I move this back